docs/sources/tutorials/send-logs-to-loki.md

```diff
@@ -93,27 +97,23 @@ services:
       - "3000:3000"
 ```
 
-Run `docker compose up` to start your Docker container and open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
+Run `docker compose up` to start your Docker container and open <http://localhost:3000> in your browser to view the Grafana UI.
 
- {{< admonition type="note" >}}
-If you the following error when you start your Docker container, `docker: 'compose' is not a docker command`, use the command `docker-compose up` to start your Docker container.
- {{< /admonition >}}
+{{< admonition type="note" >}}
+If you encounter the following error when you start your Docker container, `docker: 'compose' is not a docker command`, use the command `docker-compose up` to start your Docker container.
+{{< /admonition >}}
 
 ## Configure {{% param "PRODUCT_NAME" %}}
 
-Once the local Grafana instance is set up, the next step is to configure {{< param "PRODUCT_NAME" >}}.
+After the local Grafana instance is setup, the next step is to configure {{< param "PRODUCT_NAME" >}}.
```

Comment:
I think this should be `set up`? As I understand it `set up` is the verb and `setup` is the noun.
```suggestion
After the local Grafana instance is set up, the next step is to configure {{< param "PRODUCT_NAME" >}}.
```